Source/WebCore/dom/ContainerNode.cpp

```diff
@@ -659,7 +659,7 @@ void ContainerNode::disconnectDescendantFrames()
     disconnectSubframesIfNeeded(*this, RootAndDescendants);
 }
 
-ExceptionOr<void> ContainerNode::removeChild(Node& oldChild)
+ExceptionOr<void> ContainerNode::removeChild(const Ref<Node>& oldChild)
```

Comment:
We want `const Ref<Node>` here. 

`const Ref<Node>&` can be more efficient, but nothing prevents the referenced `Ref<Node>` from changing what it points to in surprising ways. For example, if someone called `removeChild(m_firstChild)`, something bad might happen.